src/Behat/Behat/Definition/Translator/Translator.php

```diff
@@ -4,12 +4,4 @@
 
 class Translator extends \Symfony\Component\Translation\Translator implements TranslatorInterface
 {
-    
-    public function trans($id, array $parameters = array(), $domain = null, $locale = null ) {
-        
-        if(array_key_exists('%count%', $parameters) && method_exists($this, 'transChoice') && !$this instanceof \Symfony\Contracts\Translation\TranslatorInterface){
-            return parent::transChoice($id, $parameters['%count%'], $parameters, $domain, $locale);
-        }
-        return parent::trans($id, $parameters, $domain, $locale);
-    }
 }
```

Comment:
you should probably add a deprecation for this class then

Reply:
It's still interesting to keep it as an internal stable interface I think.

See https://github.com/Behat/Behat/issues/1316